src/Microsoft.AspNetCore.Cors/Properties/AssemblyInfo.cs

```diff
@@ -3,9 +3,16 @@
 
 using System.Reflection;
 using System.Resources;
+using System.Runtime.CompilerServices;
 
 [assembly: AssemblyMetadata("Serviceable", "True")]
 [assembly: NeutralResourcesLanguage("en-US")]
 [assembly: AssemblyCompany("Microsoft Corporation.")]
 [assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
 [assembly: AssemblyProduct("Microsoft ASP.NET Core")]
+[assembly: InternalsVisibleTo("Microsoft.AspNetCore.Cors.Test,PublicKey="+
```

Comment:
Can have this all on one line (yeah, it's a super long line, but it's not a very interesting one!).